### PR TITLE
Adding Rank to ProdVersions

### PR DESCRIPTION
**Pull request by yglazko**
Trying to add a 'rank column' to Prod Versions in Signature Summary. Easy fix, good for relman.

### CODE REVIEW

**Comment by rhelmer** [comment]
Thanks! The diff here looks good to me.

Can you please squash these commits into one?
Also is there a bugzilla bug #? If so mention it in the commit message.

**Comment by yglazko** [comment]
Does this commit work? https://github.com/yglazko/socorro/commit/3a948fd4d637287c29c45fc21ae94521c4168cba

File: webapp-django/crashstats/crashstats/views.py [hunk 1]
```diff
@@ -2019,7 +2019,8 @@ def format_float(number):
             'product': r['product_name'],
             'version': r['version_string'],
             'percentage': format_float(r['percentage']),
-            'numberOfCrashes': r['report_count']})
+            'numberOfCrashes': r['report_count'],
+            'rank': r.get('currentRank', '')})
```
Comment (peterbe) [inline webapp-django/crashstats/crashstats/views.py]: I can't find a single place where this is ever present. I clicked on a bunch of signatures under Firefox 41 and it's never there. I think I need to dig deeper to try to understand what you tried achieve.

Comment (peterbe) [inline webapp-django/crashstats/crashstats/views.py]: The query that gets the data to here is this one: https://github.com/mozilla/socorro/blob/master/socorro/external/postgresql/signature_summary.py#L152-L179
No `currentRank`.

Comment (peterbe) [inline webapp-django/crashstats/crashstats/views.py]: `currentRank` is a TCBS thing (Top Crashers by Signature). You're trying to use data from the signature summary.

**Comment by peterbe** [comment]
I think this PR needs some more thought. It can't work.

**Comment by yglazko** [comment]
Closing after discussion with Kairo, going to do something with the crash volume /overall volume instead. Thank you!

**Pull request closed without merging**